lib/ohai/plugins/windows/filesystem.rb

```diff
@@ -94,6 +94,7 @@ def logical_properties(disks)
       property[:mount] = disk["name"]
       property[:fs_type] = disk["filesystem"].to_s.downcase
       property[:volume_name] = disk["volumename"].to_s.downcase
+      property[:drive_type] = disk["drivetype"].to_i
```

Comment:
@snichol123 This seems like super useful data to have. Can we get a drive_type_string value instead so that there's a human readable version of this as well? For most people seeing "2" isn't going to be real helpful unless they're familiar with the WMI classes. 

```
0 -> unknown
1 -> no_root
2 -> removable
3 -> local
4 -> network
5 -> cd
6 -> ram
```

A pretty simple helper method could return those strings when the int is passed.

Reply:
Thanks for your review @tas50. This is a good suggestion. I am not so keen on hardcoding this mapping within this plugin but from looking at the WMI API and the methods available in the Win32_LogicalDisk class there isn't one that exposes this data so I think this is the best approach to just include this within the filesystem plugin.

I have updated PR with this and also added an additional field in the output for drive type description which I think people might find useful if they are generating reports from this data.

Reply:
The output now from running this plugin looks like this:

{
  "A:": {
    "kb_size": 0,
    "kb_available": 0,
    "kb_used": 0,
    "percent_used": 0,
    "mount": "A:",
    "fs_type": "",
    "volume_name": "",
    "drive_type": 2,
    "drive_type_string": "removable",
    "drive_type_human": "3 1/2 Inch Floppy Drive"
  },
  "C:": {
    "kb_size": 107371032,
    "kb_available": 6923964,
    "kb_used": 100447068,
    "percent_used": 93,
    "mount": "C:",
    "fs_type": "ntfs",
    "volume_name": "",
    "drive_type": 3,
    "drive_type_string": "local",
    "drive_type_human": "Local Fixed Disk"
  },
  "D:": {
    "kb_size": 53684989,
    "kb_available": 13755289,
    "kb_used": 39929700,
    "percent_used": 74,
    "mount": "D:",
    "fs_type": "ntfs",
    "volume_name": "",
    "drive_type": 3,
    "drive_type_string": "local",
    "drive_type_human": "Local Fixed Disk"
  },
  "Z:": {
    "kb_size": 0,
    "kb_available": 0,
    "kb_used": 0,
    "percent_used": 0,
    "mount": "Z:",
    "fs_type": "",
    "volume_name": "",
    "drive_type": 5,
    "drive_type_string": "cd",
    "drive_type_human": "CD-ROM Disc"
  }
}